OracleTuxedo/OracleTSAM/dockerfiles/12.2.2/files/sbin/init.sh

```diff
@@ -0,0 +1,130 @@
+#!/bin/bash
+source ~/sbin/functions.sh
+source /etc/envs/bashrc
+
+nohup sudo /sbin/sshd -D >/dev/null 2>&1 &
```

Comment:
Why are you running `sshd` inside a container? How are you controlling this pid?